.ddev/commands/host/matomo_init_dev

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+## Description: Initialize Matomo environment for development
+## Usage: matomo:init:dev
+## Example: ddev matomo:init:dev
```

Comment:
Is this the split you had in mind @michalkleiner?

The "base init" should not be necessary as it is done on every startup. And,, unless my local environment acted up, testing can run without the development configuration active.

Reply:
Yep, tested and tests can be run without the dev init, matomo itself can run without both dev and tests inits so 👍